src/game.cpp

```diff
@@ -5544,3 +5550,47 @@ void Game::removeUniqueItem(uint16_t uniqueId)
 		uniqueItems.erase(it);
 	}
 }
+
+bool Game::reload(ReloadTypes_t reloadType)
+{
+	if (reloadType == RELOAD_TYPE_ACTIONS) {
+		return g_actions->reload();
+	} else if (reloadType == RELOAD_TYPE_CONFIG) {
+		return g_config.reload();
+	} else if (reloadType == RELOAD_TYPE_CREATURESCRIPTS) {
+		return g_creatureEvents->reload();
+	} else if (reloadType == RELOAD_TYPE_MONSTERS) {
+		return g_monsters.reload();
+	} else if (reloadType == RELOAD_TYPE_MOVEMENTS) {
+		return g_moveEvents->reload();
+	} else if (reloadType == RELOAD_TYPE_NPCS) {
+		Npcs::reload();
+		return true;
+	} else if (reloadType == RELOAD_TYPE_RAIDS) {
+		return g_game.raids.reload() && g_game.raids.startup();
+	} else if (reloadType == RELOAD_TYPE_SPELLS) {
+		return g_spells->reload() && g_monsters.reload();
```

Comment:
If a monster reload fails after a spells reload, you should print an error and terminate the server.